docroot/themes/contrib/civictheme/civictheme.provision.inc

```diff
@@ -160,7 +156,7 @@ function civictheme_provision(array $types = [], $clear_cache = TRUE, $verbose =
     $types = array_keys($callbacks);
   }
 
-  foreach ($callbacks as $callback_type => $callback) {
+  foreach ($callbacks as $callback_type) {
```

Comment:
was `callback_type` a key. Now it is a value which is breaking the code.

please use `foreach (array_keys($callbacks) as $callback_type) {`